packages/gatsby-theme-newrelic/src/components/CreateIssueButton.js

```diff
@@ -38,12 +39,32 @@ const CreateIssueButton = ({
     component: instrumentation?.component,
   });
 
+  const { site } = useStaticQuery(graphql`
```

Comment:
Rather than using a static query here, could we instead just make the `labels` a prop to this component?

Reply:
sure... though I may need a pointer on how do that approach. 😄

Reply:
See [this comment](https://github.com/newrelic/gatsby-theme-newrelic/pull/311#discussion_r587991463). This should provide the instructions you need 🙂

Reply:
oh I didn't even see that... ok yeah let me change the approach. at least I got practice with using static queries.